### ISSUE TITLE
unnecessary/unused line should be removed

### ISSUE THREAD

**Issue by StevePny** [NOAA-GFDL/FMScoupler#94]
This line appears unnecessary, and the returned contents of "date" are unused:
```
call fms_time_manager_get_date (Time_atmos, date(1), date(2), date(3),  &
                                 date(4), date(5), date(6))
```
https://github.com/NOAA-GFDL/FMScoupler/blob/77618869f48507c8629f28457cb701e25e1ea4fc/SHiELD/coupler_main.F90#L447C12-L447C37

Note that in FMS, get_date takes the first argument Time_atmos as an input argument:
https://github.com/NOAA-GFDL/FMS/blob/be1856c45accfe2fb15953c5f51e0d58a8816882/time_manager/time_manager.F90#L1145

The date() object is not used after being returned by the get_date routine, so this line is unnecessary and adds confusion here. (e.g. I originally assumed based on the context that "Time_atmos" was being returned and used below for subsequent checks, but instead it is defined at a higher scope.)

**Comment by bensonr** [comment]
@StevePny - Thanks for pointing this out.  It's an artifact of the lineage of this file and the introduction of intermediate restart capability from the full coupler.

**Comment by thomas-robinson** [comment]
@StevePny do you have a fix?  Can you put in a PR and link this issue?

**Comment by StevePny** [comment]
@thomas-robinson yes you should be able to just delete the line (447-448 in main):
https://github.com/NOAA-GFDL/FMScoupler/blob/93ce3642a7951eb11d7d39441911717923dfc768/SHiELD/coupler_main.F90#L447-L448

I don't have permissions to add a PR.
